zio-kafka/src/main/scala/zio/kafka/consumer/internal/Runloop.scala

```diff
@@ -184,34 +185,26 @@ private[consumer] final class Runloop private (
     else {
       for {
         consumerGroupMetadata <- getConsumerGroupMetadataIfAny
-        committableRecords = {
-          val acc             = ChunkBuilder.make[(PartitionStreamControl, Chunk[Record])](streams.size)
-          val streamsIterator = streams.iterator
-          while (streamsIterator.hasNext) {
-            val streamControl = streamsIterator.next()
-            val tp            = streamControl.tp
-            val records       = polledRecords.records(tp)
-            if (!records.isEmpty) {
-              val builder  = ChunkBuilder.make[Record](records.size())
-              val iterator = records.iterator()
-              while (iterator.hasNext) {
-                val consumerRecord = iterator.next()
-                builder +=
-                  CommittableRecord[Array[Byte], Array[Byte]](
-                    record = consumerRecord,
-                    commitHandle = commit,
-                    consumerGroupMetadata = consumerGroupMetadata
-                  )
-              }
-              acc += (streamControl -> builder.result())
-            }
-          }
-          acc.result()
-        }
-        _ <- ZIO
-               .foreachDiscard(committableRecords) { case (streamControl, records) =>
-                 streamControl.offerRecords(records)
+        _ <- ZIO.foreachDiscard(streams) { streamControl =>
```

Comment:
We should also test this with:
```suggestion
        _ <- ZIO.foreachParDiscard(streams) { streamControl =>
```

Reply:
+1